pkg/kubectl/resource/builder.go

```diff
@@ -360,13 +396,33 @@ func hasCombinedTypeArgs(args []string) (bool, error) {
 	switch {
 	case hasSlash > 0 && hasSlash == len(args):
 		return true, nil
-	case hasSlash > 0 && hasSlash != len(args):
-		return true, fmt.Errorf("when passing arguments in resource/name form, all arguments must include the resource")
```

Comment:
Running `kubectl get pods/frontend-1eni9 frontend-d0c3n` gives me `error: no resource "frontend-1eni9" is defined for group "`, as the "pods/frontend-1eni9" is seen as resource "frontend-1eni9" in group "pods". 

When the users do things like this, `hasCombinedTypeArgs` returns false (it won't reach L341), and the logic falls to the switch part -- the first arg is seen as resource and the rest args are seen as names.